cpp/spdlog/spdlog.go

```diff
@@ -0,0 +1,35 @@
+package spdlog
+
+import (
+	_ "unsafe"
+
+	"github.com/goplus/llgo/c"
+)
+
+const (
+	LLGoPackage = "link: $(pkg-config --libs cppWrap spdlog); -lcppWrap -lspdlog -pthread -lfmt"
```

Comment:
cppWrap don't need to build as a library. You can use `LLGoFiles`.

Reply:
ok, the code has changed from generating dylib to use LLGoFiles